### DISCUSSION TITLE
What are the available parameter types of a custom event?

### DISCUSSION THREAD

**Discussion by m4ttzzzz** [CodenameCrew/CodenameEngine#380] [Q&A] [answered]
What are the available parameter types? I was making a custom event that set the camera zoom.

**Comment by r6915ee** [comment, marked as answer]
Essentially, every parameter is just a variable. This means that most common types of variables are included (those being string, int, float, bool). There is also a type about the strumline, a color wheel, and a dropdown (which is just an array of selectable strings). The parameter type names, as far as I'm aware, are referred to by pascal case if you're using softcoded events.

**Reply by m4ttzzzz** [reply]
Thanks!

**Reply by NeeEoo** [reply]
<img width="702" alt="Skärmavbild 2024-09-11 kl  10 51 31" src="https://github.com/user-attachments/assets/db7459db-34fe-4854-b828-94819fd4ea5f">
